### PR TITLE
(TK-458) Prepare for 1.0.0 release

### PR DESCRIPTION
**Pull request by justinstoller**

### CODE REVIEW

**Comment by justinstoller** [comment]
We should wait to merge this until at least after I've updated the Jenkins job configs to remove Java 8.

File: MAINTAINERS [hunk 1]
```diff
@@ -4,25 +4,10 @@
   "issues": "https://tickets.puppetlabs.com/browse/TK",
   "internal_list": "https://groups.google.com/a/puppet.com/forum/?hl=en#!forum/discuss-trapperkeeper-maintainers",
   "people": [
-    {
-      "github": "aperiodic",
-      "email": "dan.lidral-porter@puppet.com",
-      "name": "Dan Lidral-Porter"
-    },
     {
       "github": "lindboe",
```
Comment (haus) [inline MAINTAINERS:8]: Congrats @lindboe you now own tk-status :)

**Review by haus — APPROVED** [review]
wash-looks-fine

**Comment by justinstoller** [comment]
https://github.com/puppetlabs/ci-job-configs/pull/3899 in the cjc change I referred to above.

File: MAINTAINERS (hunk 1 above)
Comment (lindboe) [reply MAINTAINERS:8]: finally, i have been waiting for this day

**Pull request merged**